engine/src/cmd/base_interface.cpp

```diff
@@ -1315,7 +1314,7 @@ inline QVector randyVector(float min, float max) {
 void BaseInterface::Room::Eject::Click(BaseInterface *base, float x, float y, int button, int state) {
     if (state == WS_MOUSE_UP) {
         Link::Click(base, x, y, button, state);
-        XMLSupport::parse_bool(vs_config->getVariable("physics", "AutomaticUnDock", "true"));
+        configuration()->physics.automatic_undock;  // TODO: Omit this line?
```

Comment:
yeah this looks like something that shouldn't be there as it basically doesn't do anything for the block and the value isn't saved.

Reply:
Removed